### PR TITLE
Cp/iss46

### LINKED ISSUE #46: Create a secondary toolbar
image

* Create a secondary toolbar that looks roughly like this
* Has export buttons

### PR DESCRIPTION
**Pull request by cdpadilla42**
Closes #46  Adding the export toolbar component to the project. For now, to explore the UI, it's linked to the app's navigation, but can be refactored to open from the necessary page.

### CODE REVIEW

**Comment by ryanrightmer** [comment]
I'll be reviewing this this afternoon

**Comment by cdpadilla42** [comment]
Great, thanks!

File: tenant-file/portal-app/public/index.html [hunk 1]
```diff
@@ -10,6 +10,10 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <script
```
Comment (ryanrightmer) [inline tenant-file/portal-app/public/index.html:13]: Can we set this up with NPM? https://fontawesome.com/how-to-use/on-the-web/setup/using-package-managers

File: tenant-file/portal-app/src/component/export-toolbar.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import StyledExportToolbar from './styles/ExportToolbarStyles';
+
+type Props = {
+  isExportToolsOpen: Boolean;
+};
+
+const ExportToolbar = ({ isExportToolsOpen }: Props) => {
+  return (
+    <StyledExportToolbar className={isExportToolsOpen ? '' : 'hide'}>
+      <span tabIndex={0}>
```
Comment (ryanrightmer) [inline tenant-file/portal-app/src/component/export-toolbar.tsx:11]: Can you actually make these buttons with an onClick handler? All it has to do is console.log("something") for right now, but I just want to have the hooks in place for when we need them.

**Review by ryanrightmer — COMMENTED** [review]
Looks pretty good, once you've rebased and addressed the comments, let's merge it.

**Comment by cdpadilla42** [comment]
All taken care of!

**Review by ryanrightmer — APPROVED** [review]
👍

**Pull request merged**